integration-tests/debug/src/main/resources/application.properties

```diff
@@ -0,0 +1,18 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+quarkus.native.monitoring=jmxserver,jmxclient
```

Comment:
Why not set it by default?

Reply:
I can try. There is no `BuildItem` to force it AFAIK. It could maybe be done with `SystemPropertyBuildItem`.

Reply:
Have you tried `RunTimeConfigurationDefaultBuildItem`? Maybe it is evaluated too late?

Reply:
Hmm... well `quarkus.native.monitoring` is a build time property so setting sys props wont work.

Reply:
> Have you tried RunTimeConfigurationDefaultBuildItem? Maybe it is evaluated too late?

Yes, that does not work unfortunately :disappointed:

Reply:
Ok, no problem thx for trying